setup.py

```diff
@@ -3,12 +3,15 @@
 setup(
     name='protozfitsreader',
     packages=['protozfitsreader'],
-    version='0.43.1',
+    version='0.44.0',
     description='Basic python bindings for protobuf zfits reader',
     author="Etienne Lyard et al.",
     author_email="etienne.lyard@unige.ch",
     package_data={
-        'protozfitsreader': ['*.so*'],
+        'protozfitsreader': [
+            '*.so*',
+            '*.dylib'
+        ],
         '': ['tests/resources/*'],
     },
     install_requires=['numpy', 'protobuf'],
```

Comment:
update requires. maybe read it from requirements.txt?

Reply:
regarding install_requires and requirements.txt I just read:
https://pip.pypa.io/en/stable/user_guide/#requirements-files
and 
https://caremad.io/posts/2013/07/setup-vs-requirement/

again. I fill follow what Donald Stufft wrote there.